ansible/roles/vm_set/library/vlan_port.py

```diff
@@ -47,6 +47,7 @@ def up_external_port(self):
     def create_vlan_port(self, port, vlan_id):
         vlan_port = "%s.%d" % (port, vlan_id)
         if vlan_port not in self.host_ifaces:
+            VlanPort.cmd('vconfig rem %s' % vlan_port, True)
```

Comment:
Still confused why the `vlan_port` still exists after the existence check in LINE49 failed, do you have any findings?